cfripper/config/config.py

```diff
@@ -159,3 +169,23 @@ def get_whitelisted_rules(self) -> List[str]:
                 whitelisted_rules += v
 
         return whitelisted_rules
+
+    def load_rules_config_file(self, filename: str):
+        if not os.path.exists(filename):
+            raise RuntimeError(f"{filename} doesn't exist")
+
+        try:
+            ext = os.path.splitext(filename)[1]
+            module_name = "__rules_config__"
+            if ext not in [".py", ".pyc"]:
+                raise RuntimeError("Configuration file should have a valid Python extension.")
+            spec = importlib.util.spec_from_file_location(module_name, filename)
+            module = importlib.util.module_from_spec(spec)
+            sys.modules[module_name] = module
+            spec.loader.exec_module(module)
+            rules_config = vars(module).get("RULES_CONFIG")
+            self.rules_config_schema.validate(rules_config)
+            self.rules_config = rules_config
+        except Exception:
```

Comment:
Can you catch `SchemaUnexpectedTypeError` for a slightly more specific error? Will the exception itself say what the issue with the schema is? 😄

Reply:
The logger.exception should print the information regarding that exception if I am not wrong. There is no specific action to be taken in that case. This is how the error looks like in the console:

```schema.SchemaUnexpectedTypeError: [RuleConfig(rule_mode=None, risk_value=None, filters=[Filter(reason='', eval=<function build_evaluator.<locals>.<lambda> at 0x10f972268>, rule_mode=<RuleMode.WHITELISTED: 'WHITELISTED'>, risk_value=None)])] should be instance of 'dict'```

Reply:
Looks good to me, thanks for confirming 👍